packages/marshaller/src/ddl2/model/visualization.ts

```diff
@@ -17,7 +17,7 @@ export type VizType = "Table" | "FieldForm" |
 const VizTypeMap: { [key: string]: new (...args: any[]) => {} } = {
     Table, FieldForm,
     Area, Bubble, Bar, Column, Contour, HexBin, Line, Pie, Radar, RadialBar, Scatter, Step, WordCloud,
-    USCountiesChoropleth: ChoroplethCounties, USStatesChoropleth: ChoroplethStates, ClusterPins,
+    USCountiesChoropleth: ChoroplethCounties, USStatesChoropleth: ChoroplethStates, ClusterPins: Leaflet.ClusterPins,
```

Comment:
No support for Leaflet.Circles or paths here?

Reply:
Not yet - wanted to just introduce one.  The namespace change shouldn't change anything from the Dashy point of view (apart from this line)